### ISSUE TITLE
Connection failed: [Errno 111] Connection refused to sandbox.hortonworks.com:7000

### ISSUE THREAD

**Issue by charles138** [Symantec/ambari-cassandra-service#13]
I followed your instructions to setup Cassandra to my Hortonworks Sandbox (HDP_2.6_vmware_19_04_2017_20_25_43_hdp_ambari_2_5_0_5_1
) and got the connection failed error. I googled it and found the same error for HDP 2.4 [https://community.hortonworks.com/questions/104925/ambari-cassandra-serviceconnection-failed-errno-11.html](https://community.hortonworks.com/questions/104925/ambari-cassandra-serviceconnection-failed-errno-11.html). 
Thank you in advance for your reply.

**Comment by chaimoumaCJ** [comment]
@charles138  hello Sir
Did you resolve it !
if yes , please can you help me out !

**Comment by skviknesh** [comment]
@charles138 Yes, same here! I too have this "111" issue. I am new to Big Data. I have connection issues with almost all of the services. Please, someone, help me on the same.
